src/pages/about.jsx

```diff
@@ -1,19 +1,25 @@
+import { Box, Typography } from '@mui/material'
 import Button from '@mui/material/Button'
-import Card from '@mui/material/Card'
-import CardContent from '@mui/material/CardContent'
-import CardMedia from '@mui/material/CardMedia'
-import Typography from '@mui/material/Typography'
+import Container from '@mui/material/Container'
+import Grid from '@mui/material/Grid'
+import useMediaQuery from '@mui/material/useMediaQuery'
 import Head from 'next/head'
 import Image from 'next/image'
+import { useState } from 'react'
 
 import orca from '../../public/images/orca2.png'
+import AboutCard from '../components/About/AbouCard'
+import Items from '../components/About/db.json'
 import aboutStyles from '../styles/About.module.css'
 
-export const About = () => {
+export default function About() {
+  const mobileActive = useMediaQuery('(max-width:600px)')
+  const [seeMore, setseeMore] = useState(mobileActive)
```

Comment:
```suggestion
  const [seeMore, setSeeMore] = useState(mobileActive)
```

camelCase (don't forget to update elsewhere)